site_src/routes/pages.ts

```
@@ -42,12 +42,14 @@ export function registerPageRoutes(app: Hono<AppEnv>, silverwolf: Silverwolf) {
     const nonce = c.get('nonce');
     const lv999 = c.req.query('lv') === '999';
     try {
-      const [stats, pokemonCount, marriageBenefits, poopStats, poopProfile] = await Promise.all([
+      const [stats, pokemonCount, marriageBenefits, poopStats, poopProfile, aiUsageDaily, aiUsageWeekly] = await Promise.all([
```

ESLint

[error] This line has a length of 126. Maximum allowed is 120.

(max-len)